### PR TITLE
Remove scare quotes and update tech competence words

### PR DESCRIPTION
**Pull request by TheDoubleK**
Updated words taken from comment on a buyer frontend PR here: https://github.com/alphagov/digitalmarketplace-buyer-frontend/pull/337#issuecomment-224872890

### CODE REVIEW

**Comment by superroz** [comment]
Looks good to me.

File: frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaOutcomes.yml [hunk 1]
```diff
@@ -9,11 +9,14 @@ question_advice: |
...
 
-  If you want to score individual criteria, you must add points for everything. Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
-
-  The points you give for nice-to-have skills and experience must be lower than the points you give to the supplier’s essential skills and experience and proposal.
+  You can give points to individual criteria, eg ‘have experience designing services for users with low digital
+  literacy – 15 points’. If you choose to add points, you must do it for all individual criteria.
+  
+  The points you give for nice-to-have skills and experience criteria must be lower than the points you give for 
+  essential skills and experience and proposal criteria.
+  Read [how to score individual criteria](https://www.gov.uk/guidance/how-to-set-your-evaluation-criteria-when-buying-digital-outcomes-and-specialists-services#giving-points-to-individual-criteria).
```
Comment (pcraig3) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaOutcomes.yml]: <sub>Tiny point</sub>

Do we care about "[Read how](https://github.com/alphagov/digitalmarketplace-frameworks/search?utf8=%E2%9C%93&q=%22read+how%22&type=Code)" versus "[Read about how](https://github.com/alphagov/digitalmarketplace-frameworks/search?utf8=%E2%9C%93&q=%22read+about+how%22&type=Code)"?

Comment (TheDoubleK) [inline frameworks/digital-outcomes-and-specialists/questions/briefs/technicalCompetenceCriteriaOutcomes.yml]: One for @superroz?

**Comment by allait** [comment]
👍

**Pull request merged**